test/common/upstream/eds_test.cc

```diff
@@ -1430,6 +1431,52 @@ TEST_F(EdsTest, MalformedIP) {
                             "setting cluster type to 'STRICT_DNS' or 'LOGICAL_DNS'");
 }
 
+// Test that assignment timeout is called and removes all the endpoints.
```

Comment:
Should we have a test that shows that if we have no timeout specified, that we never expire endpoints? Or is that already implicit in other tests in this file?

Reply:
We can have test to see if the time is enabled. I will add one

Reply:
Added